pip_pkg_scripts/build_pip_pkg.sh

```diff
@@ -60,6 +60,7 @@ function main() {
   rsync -avm -L  --include="*.so" --include="*_pb2.py" \
     --exclude="*.runfiles" --exclude="*_obj" --include="*/" --exclude="*" \
     bazel-bin/third_party/camera "${TMPDIR}/waymo_open_dataset"
+  touch "${TMPDIR}/waymo_open_dataset/camera/__init__.py"
```

Comment:
Maybe add an __init__.py in waymo_open_dataset/camera directly instead of creating one here?

Reply:
oh yes! should be one in `third_party/camera/__init__.py`